### ISSUE TITLE
Clippings context menu not appearing in Firefox 83 nightly

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings#304]
Firefox 83.0a1

Reported by a user via the AE Creations forums ([forum posting](https://groups.google.com/g/aecreations-help/c/Pubsf-EzOE8)):

> No right click pop-up box in today's Nightly.
Works fine in Firefox production browser 81.0.1
Using Clippings 6.2.2 in Windows 10 for both.

**Comment by aecreations** [comment]
Unable to reproduce.

Tested against Clippings 6.2.2, the current stable release. The Clippings context menu appears as normal in Firefox 83.0a1 on Windows 10. I've tested two upgrade scenarios: upgrading from Firefox 81.0.1 to Firefox Nightly 83.0a1, and from Firefox Developer Edition 82.0b9 to Firefox Nightly 83.0a1. I also tested with a new profile created in Firefox Nightly 83.0a1.  In all cases, I've observed that the Clippings context menu is appearing as expected and is fully functional.  I've also tested with Sync Clippings turned on; that is confirmed to be working as well.

I also turned on Fission, which is a new site isolation security feature that is introduced in Firefox Nightly 83 ([more info](https://wiki.mozilla.org/Project_Fission)), and Clippings continues to function normally.

**Comment by acadkins327** [comment]
I am a sweeper and use Clippings to assist in filling some forms - attached are some sample URL that you requested:
https://thebeat.us/20201014.html
https://www.bobvila.com/contest/vasagle-living-room
https://subscribe.hearstmags.com/subscribe/hgtvmagazine/260182
I have used clippings for months in all versions of Firefox Nightly and had no problems until Oct 5 update.
I wonder if it could be settings?  I did not change anything and it works in the production version which has identical settings.

Thank you for the quick response.
